app/bundles/LeadBundle/Views/Import/_list_row.html.twig

```diff
@@ -0,0 +1,67 @@
+{#
+  Variables
+    - item: Item to render
+    - actionRoute
+#}
+{% set objectRoute = null %}
+{% if 'lead' == item.object %}
+  {% set objectRoute = 'mautic_contact_index' %}
+{% endif %}
+<tr>
+    <td class="col-actions text-center">
+        <span class="label label-{{ item.satusLabelClass }}">
+            {{ ('mautic.lead.import.status.'~item.status)|trans }}
+        </span>
+    </td>
+    <td>
+        <div>
+            {% if item.status in [constant('FAILED', item), constant('IMPORTED', item), constant('MANUAL', item)] and permissions[permissionBase~':publish'] %}
```

Comment:
I think that the constants must contain the full class where the constant is defined. Otherwise the publish switch is shown all the time.
On 5.x branch:
![Screen Shot 2023-01-04 at 17 34 37](https://user-images.githubusercontent.com/1235442/210604699-6f2c9cc8-edc4-4030-ae40-3f6736d21faf.png)


On this branch:
![Screen Shot 2023-01-04 at 17 39 24](https://user-images.githubusercontent.com/1235442/210604812-88cbb5bd-42ba-4771-8aad-5bf67f15148a.png)